package.json

```diff
@@ -13,7 +13,7 @@
     "@wireapp/react-ui-kit": "9.28.0",
     "@wireapp/store-engine-dexie": "2.1.15",
     "@wireapp/telemetry": "0.1.4",
-    "@wireapp/webapp-events": "0.24.3",
+    "@wireapp/webapp-events": "file:.yalc/@wireapp/webapp-events",
```

Comment:
To be replaced after https://github.com/wireapp/wire-web-packages/pull/6750 is merged